### PR TITLE
Enh/GitHub webhook logging

### PR DESCRIPTION
**Pull request by sven-petersen**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

Wait for https://github.com/gardener/dashboard/pull/1454

**Release note**:

```feature operator

```

### CODE REVIEW

**Comment by gardener-robot-ci-1** [comment]
Thank you @sven-petersen for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

File: backend/lib/github/SyncManager.js [hunk 1]
```diff
@@ -56,6 +58,7 @@ class SyncManager {
   async #throttledLoad () {
     clearTimeout(this.#scheduledInvocationTimeoutId)
     const wait = Math.max(0, this.#lastInvokeTime + this.#throttle - Date.now())
+    wait && logger.debug(`GitHub synchronization delayed due to throttling for ${wait / 1000}s`)
```
Comment (holgerkoser) [inline backend/lib/github/SyncManager.js:61]:
```suggestion
    if (wait) {
      logger.debug(`GitHub synchronization delayed due to throttling for ${wait / 1000}s`)
    }
```

**Review by petersutter — APPROVED** [review]
/lgtm

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Pull request merged**